src/network/socketFactory.js

```diff
@@ -7,7 +7,7 @@ module.exports = () => {
   const net = require('net')
   const tls = require('tls')
 
-  return ({ host, port, ssl, onConnect }) => {
+  return ({ id, host, port, ssl, onConnect }) => {
```

Comment:
It looks like this just gets dropped here. @fbonecco-cribl  do we want to do anything with this (e.g. decorate the created Socket?

Reply:
Correct. We're not using the id here, but in Stream: https://bitbucket.org/cribl/cribl/src/205516e4fa6c5dd77c3eb009036a4aa03e3e38cf/src/sluice/js/input/kafka/KafkaConnection.ts#lines-138,13

Reply:
Aaaah , right :man_facepalming: gotcha. Then looks good to me.